src/Microsoft.AspNetCore.Mvc.Core/ObjectResult.cs

```diff
@@ -29,6 +29,7 @@ public ObjectResult(object value)
         /// <summary>
         /// Gets or sets the HTTP status code.
         /// </summary>
+        [StatusCodeValue]
         public int? StatusCode { get; set; }
```

Comment:
What about the other properties that implement `IStatusCodeActionResult`? Wouldn't it be nicer to check if a property implements the interface instead?

Reply:
Done. Switched to using the interface contract to compare, forgot all about it